### PR TITLE
Increase test Coverage of pkg/authorization

### PR DESCRIPTION
**Pull request by raja-sami-10p**
Signed-off-by: Raja Sami <raja.sami@tenpearls.com>

**- What I did**
Increase unit test coverage of pkg/authorization

**- How to verify it**
```sh
make test-unit
```

**- A picture of a cute animal (not mandatory but encouraged)**
:hamster:

### CODE REVIEW

**Review by boaz0 — COMMENTED** [review]
Running this unit-test on Windows returns these errors:
```
pkg\authorization\middleware_test.go:25: undefined: authZPluginTestServer

pkg\authorization\middleware_test.go:29: undefined: createTestPlugin
```

**Comment by raja-sami-10p** [comment]
@ripcurld0,
I've change func TestMiddlewareWrapHandler to skip for windows environment.

File: pkg/authorization/middleware_unix_test.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	server.start()
+	defer server.stop()
+
+	authZPlugin := createTestPlugin(t)
+	pluginNames := []string{authZPlugin.name}
+
+	var pluginGetter plugingetter.PluginGetter
+	middleWare := NewMiddleware(pluginNames, pluginGetter)
+	handler := func(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+
```
Comment (AkihiroSuda) [inline pkg/authorization/middleware_unix_test.go]: nit: extra line

File: pkg/authorization/middleware_unix_test.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		return nil
+	}
+
+	authList := []Plugin{authZPlugin}
+	middleWare.SetPlugins([]string{"My Test Plugin"})
+	setAuthzPlugins(middleWare, authList)
+	mdHandler := middleWare.WrapHandler(handler)
+	require.NotNil(t, mdHandler)
+
+	addr := "www.authz.com/auth"
```
Comment (AkihiroSuda) [inline pkg/authorization/middleware_unix_test.go]: nit: `example.com` is preferred over `authz.com` (available for sale!), though it does not matter in this case (httptest)

**Review by AkihiroSuda — APPROVED** [review]
LGTM

File: pkg/authorization/middleware_unix_test.go (hunk 2 above)
Comment (raja-sami-10p) [inline pkg/authorization/middleware_unix_test.go]: changed

File: pkg/authorization/middleware_unix_test.go (hunk 1 above)
Comment (raja-sami-10p) [inline pkg/authorization/middleware_unix_test.go]: changed

**Comment by vieux** [comment]
LGTM

**Pull request merged**